test/browser/webdriver-sauce.conf.js

```diff
@@ -147,7 +159,7 @@ exports.config = {
   // See the full list at http://mochajs.org/
   mochaOpts: {
     ui: 'bdd',
-    timeout: 60000
+    timeout: 120000
```

Comment:
Why more?

Reply:
While watching the Sauce Labs replay videos I noticed it can take browsers awhile to load the giant ~ 4MB netlify-cms file so I figured we should give them plenty of time.